sdk/src/main/java/io/dapr/client/DaprClient.java

```diff
@@ -78,44 +106,44 @@
   /**
    * Retrieve a State based on their key.
    *
-   * @param state        The key of the State to be retrieved
+   * @param state        The key of the State to be retrieved.
    * @param stateOptions
    * @param clazz        the Type of State needed as return.
-   * @param <T>          the Type of the return
-   * @param <K>          The Type of the key of the State
-   * @return A Mono Plan for the requested State
+   * @param <T>          the Type of the return.
+   * @param <K>          The Type of the key of the State.
+   * @return A Mono Plan for the requested State.
    */
   <T, K> Mono<T> getState(StateKeyValue<K> state, StateOptions stateOptions, Class<T> clazz);
 
   /**
    * Save/Update a list of states.
    *
-   * @param states  the States to be saved
-   * @param options the Options to use for each state
-   * @param <T>     the Type of the State
-   * @return a Mono plan of type Void
+   * @param states  the States to be saved.
+   * @param options the Options to use for each state.
+   * @param <T>     the Type of the State.
```

Comment:
For bulk save state, T can be different for each state. Similarly StateOptions can be different for each state. This api signature needs to be revisited.

Reply:
For these state apis, we would need metadata (it could be a map)

Reply:
Yes, this is not the scope of this change.